site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -33,7 +38,8 @@ This includes NPM which is also needed. Currently version 10.11.0 is recommended
     different node versions which might be lower than 10.11.0 and may cause problems)
 -   Install using NVM by following the instructions [here](https://github.com/creationix/nvm#install-script)
 -   Download and install the package from the [NodeJS website](https://nodejs.org/en/)
-
+##### Windows10
+-   Download and install the package from the [NodeJS website](https://nodejs.org/en/)
 #### Install [Watchman](https://facebook.github.io/watchman/). (minimum required version is 4.9.0)
```

Comment:
```suggestion
#### Install [Watchman](https://facebook.github.io/watchman/)

The minimum required version is 4.9.0.
```

Reply:
Is https://facebook.github.io/watchman/ the correct URL? The URL is mentioned again but in a different order further down the document.

Reply:
@justinegeffen yes that is the main domain address. Mac can use Homebrew to install Watchman. Linux must install it from source. And I added the instruction for Windows 10 which uses beta package.